### ISSUE TITLE
Rethinking Image Deraining via Rain Streaks and Vapors 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#51]
# TL;DR
Conventional raindrop removal methods have been unsuccessful due to water vapor around raindrops and contrast changes in the entire image. In this study, the researchers proposed a mechanism to pre-train the network to predict raindrop removal and contrast change, and then train it to include water vapor removal as well. The removal of raindrops was cleaner than in previous studies.
![image](https://user-images.githubusercontent.com/24524018/90889207-a818a680-e3f2-11ea-9ecf-b37ebcf2d252.png)
![image](https://user-images.githubusercontent.com/24524018/90889215-ab139700-e3f2-11ea-9810-152fdeb3cc5a.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2008.00823

# Submission Dates(yyyy/mm/dd)

# Authors and institutions
Yinglong Wang, Yibing Song, Chao Ma, Bing Zeng
- University of Electronic Science and Technology of China
- Tencent AI Lab
- MoE Key Lab of Artificial Intelligence, AI Institute, Shanghai Jiao Tong University 

# Methods

# Results

# Comments
